mifosng-android/src/main/java/com/mifos/mifosxdroid/offline/offlinedashbarod/OfflineDashboardFragment.java

```diff
@@ -66,16 +87,19 @@
     OfflineDashboardAdapter mOfflineDashboardAdapter;
 
     // update mPayloadIndex to number of request is going to fetch data in Presenter;
-    private int mPayloadIndex = 2;
+    private int mPayloadIndex = 3;
 
     private static final int GRID_COUNT = 2;
 
     private List<Class> mPayloadClasses;
 
+    public static final String[] SYNC_CARD_UI_NAMES = {"Sync Clients",
```

Comment:
If these names are shown to the user, they must be accessed from String-Array Resource